src/parser/mod.rs

```diff
@@ -3534,7 +3534,9 @@ impl<'a> Parser<'a> {
     pub fn parse_all_or_distinct(&mut self) -> Result<Option<Distinct>, ParserError> {
         let loc = self.peek_token().location;
         let all = self.parse_keyword(Keyword::ALL);
-        let distinct = self.parse_keyword(Keyword::DISTINCT);
+        let distinct = self
+            .parse_one_of_keywords(&[Keyword::DISTINCT, Keyword::DISTINCTROW])
+            .is_some();
```

Comment:
@yoavcloud this one is unadressed? (thinking the current behavior of repurposing distinct for distinctrow isn't ideal since they're not necessarily the same flag)

Reply:
I decided it's best to remove this from the PR, as it's unrelated and I have not done enough research on it. In MySQL, DISTINCTROW and DISTINCT are synonymous but that's not true across all dialects. I'll add it in a future PR.

Reply:
Alright that sounds good! We can remove the code for it (we have the keyword and the associated parsing on this line)?

Reply:
I forgot to push my last commit, it's there now